src/module/system/localize.ts

```diff
@@ -8,20 +7,4 @@ type TranslationsPF2e = Record<string, TranslationDictionaryValue> &
     typeof ActionTranslations &
     typeof RETranslations;
 
-export class LocalizePF2e {
-    static ready = false;
-
-    private static _translations: TranslationsPF2e;
-
-    static get translations(): TranslationsPF2e {
-        if (!this.ready) {
-            throw ErrorPF2e("LocalizePF2e instantiated too early");
-        }
-        if (this._translations === undefined) {
-            this._translations = mergeObject(game.i18n._fallback, game.i18n.translations, {
-                enforceTypes: true,
-            }) as TranslationsPF2e;
-        }
-        return this._translations;
-    }
-}
+export { TranslationsPF2e };
```

Comment:
Could this be moved to an existing file?

Reply:
It's not really used anywhere but eslint complains if one of the json files isn't imported. 

![image](https://user-images.githubusercontent.com/41452412/233808354-01d99423-654b-4bbf-b7f2-6c5575f980ec.png)

Reply:
If `TranslationsPF2e` isn't used anywhere, can be kill the whole file?

Reply:
Yeah, if I can figure out how to appease eslint,

Reply:
Push it up with eslint complaining

Reply:
Done